openslides_backend/action/actions/user/create_update_permissions_mixin.py

```diff
@@ -268,59 +268,6 @@ def check_group_C(
             touch_meeting_ids: Set[int] = set(
                 map(int, instance.get("group_$_ids", dict()).keys())
```

Comment:
This is the basis idea: All meetings touched by group are mentioned here and the user has to have the rights (OML, CML for the meeting or meeting) to fulfill. I expect all cases of adding or removing groups to the user are covered with this expression.